docs/deployment/deployPACluster.md

```diff
@@ -0,0 +1,169 @@
+---
+# Deploy a PingAccess Cluster Locally Without a Server Profile
+
+!!! warning "Demo Use Only"
+    The instructions in this document are for testing and learning and are not intended for use in production.
+
+## Purpose
+Create and deploy a default PingAccess Cluster, without having to create a custom server profile. This process will allow you to quickly bring up the PingAccess UI and conduct any tests you need.
+
+## Prerequisites
+
+* [kubectl](https://kubernetes.io/docs/tasks/tools/#kubectl)
+* Access to a Kubernetes cluster
+* Access to EKS AWS image repository and the image tags
+
+## Steps
+1. Confirm that your kuberenetes context and namespace are set correctly
+
+    ```sh
+    # Display kuberenetes context
+    kubectx
+
+    # Display namespace
+    kubens -c
+    ```
+
+    1. If these values are not set or are incorrect, you can set them with the following commands. If you do not yet have a namespace, or do not have access to a kubernetes cluster, refer to [Deploy Example Stack](https://devops.pingidentity.com/get-started/getStartedExample/).
+
+        ```sh
+        # Display kuberenetes context
+        kubectx ${context}
+
+        # Display namespace
+        kubens ${namespace}
+        ```
+1. Confirm that there are no persistent volumes
+
+    ```sh
+    #List any persistent volumes
+    kubectl get pvc
+
+    #Delete name_of_pvc persistent volume
+    kubectl delete pvc <name_of_pvc>
+    ```
+
+1. Create a YAML file similar to the one shown here. Make sure to replace "docker-builds-eks-repo" and "tag_name" with the repository and tag from which you wish to pull and build.
+
+    ```sh
+    global:
+    envs:
+        PING_IDENTITY_ACCEPT_EULA: "YES"
+    ingress:
+        enabled: true
+        addReleaseNameToHost: prepend
+        defaultDomain: ping-devops.com
+        annotations:
+        nginx.ingress.kubernetes.io/backend-protocol: "HTTPS"
+        kubernetes.io/ingress.class: "nginx-public"
+
+    #############################################################
+    # pingaccess-admin values
+    #############################################################
+    pingaccess-admin:
+    enabled: true
+    privateCert:
+        generate: true
+    image:
+        repositoryFqn: ${docker-builds-eks-repo}/pingaccess
+        tag: ${tag_name}
+    envs: 
+        PING_IDENTITY_PASSWORD: "2FederateM0re!"
+
+    #############################################################
+    # pingaccess-engine values
+    #############################################################
+    pingaccess-engine:
+    enabled: true
+    container:
+        replicaCount: 1
+    image:
+        repositoryFqn: ${docker-builds-eks-repo}/pingaccess
+        tag: ${tag_name}
+    envs: 
+        PING_IDENTITY_PASSWORD: "2FederateM0re!"
+    ```
+
+1. Create the default PingAccess cluster. Make sure that you fill in the "PATH" to your new values.yaml file. This deployment may take a few minutes to become healthy.
```

Comment:
Consider using a different variable name here other than `PATH`, to avoid confusion with the shell built-in variable `PATH`